vault/managed_key.go

```diff
@@ -0,0 +1,11 @@
+package vault
```

Comment:
Should this have a negative enterprise build flag? Or will we remove these definitions from the enterprise side?

Reply:
The latter